Answers.md

```diff
@@ -2,10 +2,20 @@
 
 1.  What is React JS and what problems does it try and solve?
 
+    React JS try to ease the pain of directly manipulate DOM elements during web development

```

Comment:
Great answer! React also helps us with reusability and scalability due to its component-based structure and the virtual DOM